go.mod

```diff
@@ -6,6 +6,7 @@ replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
 	k8s.io/client-go => k8s.io/client-go v0.18.0
+	launchpad.net/gocheck => github.com/go-check/check v0.0.0-20200227125254-8fa46927fb4f
```

Comment:
Why is this replace here?

Reply:
I was getting errors with this library updating go modules, but it seems not needed now. Removed